pkgs/by-name/pi/pixelflasher/package.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  android-tools,
+  cacert,
+  fetchFromGitHub,
+  makeDesktopItem,
+  makeWrapper,
+  python311,
+  stdenv,
+  substituteAll,
+}:
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
I don't think `buildPythonApplication` works with `pyinstaller`, but I'd love to be shown otherwise.

Reply:
I'm not experienced enough with python or packaging it to say either way, so I'm leaving this to the next reviewer/committer. Otherwise everything LGTM, nice work.